explorer/app/components/Layout/components/Header/components/Content/content.tsx

```diff
@@ -4,20 +4,20 @@ import { HEADER_HEIGHT } from "../../header";
 
 interface Props {
   children: ReactNode | ReactNode[];
-  desktop: boolean;
+  desktopSm: boolean;
```

Comment:
All changes from `desktop` to `desktopSm`, here and elsewhere, are a change in breakpoint media query for the header.